examples/default/copies.json

```diff
@@ -0,0 +1,47 @@
+{
+  "@context": "https://github.com/Inria-Visages/blob/master/BIDS-prov/context.json",
+  "@id": "http://example.org/ds00000X",
+  "generatedAt": "2020-03-10T10:00:00",
+  "wasGeneratedBy": {
+      "@id": "INRIA",
+      "@type": "Project",
+      "startedAt": "2016-09-01T10:00:00",
+      "wasAssociatedWith": { "@id": "NIH",
+                              "@type": "Organization",
+                              "hadRole": "Funding"
+                            }
+    },
+  "@graph": [
+    {
+      "prov:Activity" : [
+        {
+          "@id" : "copy",
+          "wasAssociatedWith" : {
+            "@type" : "prov:SoftwareAgent",
+            "RRID" : "https://scicrunch.org/resources/Any/record/nlx_144509-1/SCR_007037/resolver?q=SPM&l=SPM",
+            "label" : "SPM" 
+          },
+          "startedAtTime" : "2019-10-10T10:00:00",
+          "endedAtTime" : "2019-10-10T10:00:00",
+          "used" : "$HOME/BIDS-prov/nidm-results-examples/spm_default/ds011/sub-01/func/sub-01_task-tonecounting_bold.nii.gz"
+        }
+      ],
+      "prov:Entity" : [
+        {
+          "@id": "$HOME/BIDS-prov/nidm-results-examples/spm_default/ds011/sub-01/func/sub-01_task-tonecounting_bold.nii.gz",
+          "@type": "bids:gzip_file",
```

Comment:
and @cmaumet I still have this `bids.gzip_file` type left 
Do i simply remove it ? Is there any benefit to providing subtype information like this ?

Reply:
For now I would say let's stick to prov:Entity (until we know more about what is currently defined in the BIDS namespace).

Reply:
You can therefore also remove 'bids' from the context (and that's good because we are not sure yet what is the URI we should use for it)